### PR TITLE
Add buttercms w/ npm auto-update

### PR DESCRIPTION
**Pull request by ViolanteCodes**
add buttercms w/npm auto-update

### CODE REVIEW

File: packages/b/buttercms.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          "basePath": "lib",
+          "files": [
+            "*.js",
+            "utils/*.js",
+            "resources/*.js"
+          ]
+        }
+      ]
+    },
+    "author": "ButterCMS <support@buttercms.com>"
```
Comment (MattIPv4) [inline packages/b/buttercms.json:34]: As the CI notes, we don't accept an `author` property, but we do accept an `authors` property that is an array of objects with a `name`/`email` property.

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/b/buttercms.json (hunk 1 above)
Comment (ViolanteCodes) [reply packages/b/buttercms.json]: > As the CI notes, we don't accept an `author` property, but we do accept an `authors` property that is an array of objects with a `name`/`email` property.

@MattIPv4 Sorry about that. I have fixed and squashed. Thanks!

File: packages/b/buttercms.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "ButterCMS",
+  "description": "ButterCMS JS API Client",
+  "filename": "buttercms.min.js",
```
Comment (MattIPv4) [inline packages/b/buttercms.json:4]: As the CI error notes, the filename you've set here does not exist in the package files. I suspect you want `butter.js`?

File: packages/b/buttercms.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      }
+    ]
+  },
+  "authors": [
+    {
+      "name": "ButterCMS",
+      "email": "support@buttercms.com"
+    }
+  ]
+}
```
Comment (MattIPv4) [inline packages/b/buttercms.json:40]:
```suggestion
}

```
Newline eof please

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/b/buttercms.json (hunk 2 above)
Comment (ViolanteCodes) [reply packages/b/buttercms.json]: :grimacing: I have updated, sorry about that.

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Comment by ViolanteCodes** [comment]
@MattIPv4 Hello - just a question: How long does this normally take to appear on the website? We'd like to update our cdn links to include the new cdnjs links, but have not been able to find the butter package. Thank you! :smile:

**Comment by MattIPv4** [comment]
@xtuc might need your eyes here -- unless I'm blind, this config is valid and exists in this repo, yet no logs exist for it at all?

**Comment by ViolanteCodes** [comment]
@MattIPv4 Hello - just checking in on this to see if there's anything you need on our end, thank you.

**Comment by MattIPv4** [comment]
Nope, this is blocked on some sort of bug that I am awaiting @xtuc to investigate on the Cloudflare side.

**Pull request merged**